web/gerbera.html

```diff
@@ -23,9 +23,9 @@
         </div>
         <div class="collapse navbar-collapse">
             <ul class="nav navbar-nav navbar-left">
-                <li><a id="nav-home" class="navbar-link" data-gerbera-menu-cmd="HOME">Home <span class="sr-only">(current)</span></a></li>
-                <li><a id="nav-db" class="navbar-link disabled" data-gerbera-menu-cmd="SELECT_TYPE" data-gerbera-type="db">Database</a></li>
-                <li><a id="nav-fs" class="navbar-link disabled" data-gerbera-menu-cmd="SELECT_TYPE" data-gerbera-type="fs">File System</a></li>
+                <li><a id="nav-home" class="btn navbar-link" data-gerbera-menu-cmd="HOME">Home <span class="sr-only">(current)</span></a></li>
```

Comment:
Adding the `btn` class makes the `disabled` class work properly.  Previously, `disabled` was not getting applied.